src/Numberdrome.js

```diff
@@ -1,7 +1,39 @@
 'use strict';
 
 class Numberdrome {
-  // write code here
+  constructor() {
+    this.list = [];
+  }
+
+  addNumber(n) {
+    this.list.push(n);
+  }
+
+  removeNumber(n) {
+    this.list = this.list.filter(number => number !== n);
+  }
+
+  sum() {
+    return this.list.reduce(
+      (accumulator, currentNumber) => accumulator + currentNumber, 0);
```

Comment:
некорректный перенос, просто обрыв строки
```suggestion
    return this.list.reduce((accumulator, currentNumber) => (
      accumulator + currentNumber
    ), 0);
```